packages/api-client/src/views/Request/RequestSection/RequestTable.vue

```diff
@@ -8,6 +8,7 @@ import { ScalarButton, ScalarIcon } from '@scalar/components'
 import type { RequestExampleParameter } from '@scalar/oas-utils/entities/spec'
 import { computed } from 'vue'
 
+import { hasItemProperties } from '../libs/request'
```

Comment:
can we use the @/ alias here?

Reply:
only if its needed/used elsewhere. if its only used here then its fine here.